### ISSUE TITLE
glew-config.cmake not found for ubuntu 18.04

### ISSUE THREAD

**Issue by arrenglover** [robotology/superimpose-mesh-lib#9]
The glew-config.cmake is not installed with the default package in ubuntu 18.04  i.e. using `sudo apt install libglew-dev`

therefore cmake cannot find the file and fails:
```
CMake Error at src/SuperimposeMesh/CMakeLists.txt:22 (find_package):
Could not find a package configuration file provided by "glew" with any of  the following names:
glewConfig.cmake
glew-config.cmake 
```

A workaround is to install the package from source or use an alternative package management, i.e. `usr/lib/x86_64-linux-gnu/pkgconfig/glew.pc ` is installed.

**Comment by claudiofantacci** [comment]
A possible solution is to use `pkg_config` if `find_package` fails to fin GLEW and then use ([StandardFindModule.cmake](https://github.com/robotology/ycm/blob/master/modules/StandardFindModule.cmake) from [YCM](https://github.com/robotology/ycm)) to create a target for the library.

**Comment by claudiofantacci** [comment]
To address this issue, I had to add [YCM](https://github.com/robotology/ycm) as a new dependency (see https://github.com/robotology/superimpose-mesh-lib/commit/5dd094048e982162d09997a018f378ff68331624). While this may seems to be yet another dependencies, YCM simply install some advanced and useful CMake modules and scripts (no compilation is required, it's a plain installation) that allows to maintain libraries easily and to keep the pace with upstream CMake on those systems with older CMake.
Furthermore, I fixed the CMake's `FindGLEW` [upstream](https://gitlab.kitware.com/cmake/cmake/merge_requests/3094) so as to simplify it's usage within YCM as soon as possible.

Long story short: as soon as `FindGLEW` is merged [upstream](https://gitlab.kitware.com/cmake/cmake/merge_requests/3094), we include it in YCM and we solve this issue once and for all on all systems.

I'll close this issue as soon as this happens.

**Comment by claudiofantacci** [comment]
The new version of the library now makes use and require dependency [YCM](https://github.com/robotology/ycm). YCM helps to ship updated CMake files faster either new one or from future CMake releases. The current stable version of YCM provides, [0.10.2](https://github.com/robotology/ycm/releases/tag/v0.10.2), provides an all-new `FindGLEW.cmake` downloaded from upstream CMake that fixes this issue. Please use devel version `0.10.100` or newer to have this issue fixed.
